### PR TITLE
Threatstream: Feature - Add 3 new flags to work with ip/domain reputation

### PR DESCRIPTION
**Pull request by paulnawat**
Please ensure your pull request (PR) adheres to the following guidelines:

- Please refer to our contributing documentation for any questions on submitting a pull request, link: [Contribution Guide](https://github.com/splunk-soar-connectors/.github/blob/main/.github/CONTRIBUTING.md)

#### Please check if your PR fulfills the following requirements:
- [X] Testing of all the changes has been performed (for bug fixes / features)
- [X] The readme.html has been reviewed and added / updated if needed (for bug fixes / features)
- [X] Use the following format for the PR description: `<App Name>: <PR Type> - <PR Description>`
- [X] Provide release notes as part of the PR submission which describe high level points about the changes for the upcoming GA release.
- [X] Verify all checks are passing.
- [X] Do NOT use the `next` branch of the forked repo. Create separate feature branch for raising the PR.
- [X] Do NOT submit updates to dependencies unless it fixes an issue.
- [X] Limit your pull request to one application and submit multiple pull requests if necessary.
#### Please check the type of change your PR introduces:
- [ ] New App
- [ ] Bugfix
- [X] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Documentation
- [ ] Other (please describe): 

## Release Notes (REQUIRED)
* Add 3 new flags to work with "ip reputation" and "domain reputation" to enable fetching partial info.

## What is the current behavior? (OPTIONAL)
- ip/domain reputations are always returned in full.

## What is the new behavior? (OPTIONAL)
- allow fetching partial reputation.
---
Thanks for contributing!

### CODE REVIEW

File: threatstream.json [hunk 1]
```diff
@@ -495,6 +495,21 @@
                     "description": "Fetch extra data from Anomali server if available",
                     "data_type": "boolean",
                     "order": 2
+                },
+                "pdns": {
+                    "description": "If enabled, Fetch pdns",
```
Comment (tonyc-phantom) [inline threatstream.json:500]: Should "fetch" be capitalized in these new parameter descriptions?

File: threatstream.json [hunk 2]
```diff
@@ -819,6 +858,21 @@
...
```
Comment (tonyc-phantom) [inline threatstream.json:863]: Same here

**Review by tonyc-phantom — DISMISSED** [review]

File: threatstream.json (hunk 1 above)
Comment (paulnawat) [reply threatstream.json:500]: oh good point. What do you think about "If enabled, pdns will also be fetched"?

Comment (tonyc-phantom) [reply threatstream.json:500]: sure

Comment (paulnawat) [reply threatstream.json]: super. I will also update `insights` and `external referecnes`

File: threatstream.json (hunk 2 above)
Comment (paulnawat) [reply threatstream.json]: updated as discussed.

**Review by brian-phantom — APPROVED** [review]

**Review by joshquerinphantom — APPROVED** [review]

**Comment by paulnawat** [comment]
@joshquerinphantom  as discussed, the lint error will be fixed in a separate ticket PAPP-23198

**Pull request merged**